gcc.spec

```diff
@@ -289,3 +335,23 @@ find %{i}/lib %{i}/lib64 -name '*.la' -exec rm -f {} \; || true
 %define keep_archives yes
 # This avoids having a dependency on the system pkg-config.
 rm -rf %{i}/lib/pkg-config
+
+%post
+
+%ifos darwin
+dirs=`find $CMS_INSTALL_PREFIX/%{pkgrel} -name libgcc.a | head -1 | xargs dirname `
+if [ -n "${dirs}" ]
+then
+$CMS_INSTALL_PREFIX/%{pkgrel}/bin/gcc -dumpspecs >${dirs}/specs
+perl -p -i -e 'use Env qw(CMS_INSTALL_PREFIX); s|\*link:|\*link: -rpath $CMS_INSTALL_PREFIX/%{cmsplatf} -headerpad_max_install_names \\|g' ${dirs}/specs
+fi
+for x in `find $CMS_INSTALL_PREFIX/%{pkgrel}/ -type f | grep -v -e "[.]pyc" | grep -v libgcc_ext.10 `; do 
+    filestr=`file $x | cut -d: -f2 | tail -1`
+    if [ -z "${filestr##*Mach-O*}" ];then
+      chmod +w $x
+      install_name_tool -add_rpath  $CMS_INSTALL_PREFIX/%{cmsplatf} $x || true
```

Comment:
@smuzaffar This is where you can add -headerpad_max_install_names to the gcc spec so that Mach-o binaries have enough space to change rpath and id